### ISSUE TITLE
Excel macro for converting spreadsheets from REFPROP 9.x to 10

### ISSUE THREAD

**Issue by WeilandN** [usnistgov/REFPROP-wrappers#107]
Per the Welcome page documentation in REFPROP.xls, the REFPROP.xla macro for converting existing Excel files with REFPROP 9.x or earlier function calls to the new REFPROP 10 format does not appear when trying to View Macros in Excel.  Per the documentation, "If the macro doesn't exist, go to the VBA code (press Alt-F11), then to the ConvertRefprop9to10 section, and change the first line from "Function" to "Sub". "  Even with this change, the macros would not show up for me in my version of Excel.  I tried several other methods (e.g., saving as a REFPROP.xlam file and using that for the add-in), but without success.  

As a workaround, I copied the ConvertRefprop9to10 Module in VBA under REFPROPExcelSpreadsheet (REFPROP.xla) to the Modules section of my spreadsheet, which allowed me to see and run the macro.  Once done, the Module could be deleted from my spreadsheet.  If there's another solution to enable users to see and run the macros, please relay this.

Three issues with the macro:
1. The macro needs to be run individually for each tab in the spreadsheet that contains REFPROP function calls.  It would be nice if the macro updated the entire workbook instead.
2. The sequencing of the statements in the macro needs to be updated to move the search terms with prefixes above those with just the root terms.  For instance, the macro will replace the formula =LiquidDensity("water","Tliq","SI with C", 20)  with =LiquidREFPROP("D","water","Tliq","SI with C", 20), since the search term for "Density(" precedes that of "LiquidDensity(" in the macro code.  Reordering all of the search terms allowed the macro to work properly.  This is a suggested improvement to the patch REFPROP.xls and REFPROP.xla files that have been posted, in addition to the suggestion to enable the macro to update the entire workbook.
3. If you could, add hyperlinks to the "https://trc.nist.gov/refprop/10-PATCH/REFPROP.XLS" and "https://trc.nist.gov/refprop/10-PATCH/REFPROP.XLA" mentions on the "Resolving Problems Linking with Excel" section of the NIST REFPROP FAQ webpage - this makes it much easier to download the file by right clicking the link and clicking "Save Target As..." 

Thanks!

**Comment by EricLemmon** [comment]
I've updated the Excel section on the FAQ page with your comments, and included links to the new Excel files with the changes made that you suggested.  

I'm not sure how to make Excel run the macro for every sheet in your file; thus that has not been fixed.  I also added the comments about your workaround to make the macros work so that others can see what you did.

The FAQ link is:  https://pages.nist.gov/REFPROP-docs/
